animation-frame-rate/README.md

```diff
@@ -0,0 +1,128 @@
+
+Devices from other manufacturers also have screen refresh rates that are greater than 60Hz; mobile devices exist with 90Hz and 120Hz refresh rates, and desktop screens exist with 144Hz and 240Hz refresh rates.
+
+On Apple platforms, accelerated animations will already run at 120Hz thanks to Core Animation's built-in support for the ProMotion technology, whereas the rest of the Web page only updates at 60Hz. This "rest of the Web page" includes script-driven animations using `requestAnimationFrame()` as well as all non-accelerated Web Animations (including CSS Transitions and CSS Animations).
+
+WebKit chose to restrict web content updates to 60Hz for two reasons: first, anything faster would trigger a significant increase in power usage, and second, we found several examples of web pages that had incorrect behavior when `requestAnimationFrame()` callbacks were fired at a non-60Hz frequency.
+
+However, now that there are more devices with high refresh rate displays, we believe it's judicious to propose advancing the Web platform to take advantage of higher refresh rates and allow authors to choose a suitable refresh rate for different kinds of animations.
+
+## Variable frame rate
+
+Most high refresh-rate displays actually use variable refresh rates; for example, a 120Hz display might be able to update at 120Hz, 60Hz, 30Hz, etc. This frequency adaptation is responsive to content being displayed on screen, for instance it's optimal to refresh the display at 24Hz playing a 24fps video in fullscreen with no visible UI.
+
+But the frame rate variation may be dictated by other factors than just the content. For instance, the system may impose frame rate limits when in low power mode, or when the target frame rate has not been achieved or some period of time.
+
+Generally, it is important to account for the power cost of updating at a high frequency, and driving the display at its maximum frequency is not something that should be encouraged unless it helps fulfilling compelling use cases on the web.
+
+## Use cases
+
+We think that it's important that API enhancements in this area allow authors to communicate the need for both higher and lower-frequency animations. There is generally two categories we see where the authors would want to specify an intent to update at a rate different than the default.
+
+**Higher Frequency**
+
+* WebGL or 2D canvas game or visualization
+* High-impact CSS/SVG/DOM animation
+
+**Lower Frequency**
+
+* Animation on a small element (e.g. progress meter)
+* Background ambient animation
+* Fade animations (opacity, color, some filters) where lower frame rates are less noticeable
```

Comment:
* Animated content that uses discrete steps?

Reply:
Addressed in 0625b55.